portfolio/src/main/webapp/script.js

```diff
@@ -30,8 +30,8 @@ function addRandomLanguage() {
 /**
  * Appends previously made comments that populated the servlet as children list elements to the page.
  */
-function showComments() {
-  fetch('/data').then(response => response.json()).then((commentsJson) => {
+function showComments(val) {
+  fetch('/data?max='+val).then(response => response.json()).then((commentsJson) => {
```

Comment:
Same here: make sure you add a space between operators and variable names (so, '/data?max=' + val)

Reply:
Ack!